File: src/main/java/net/mjduffin/risk/usecase/request/AttackRequest.java
package net.mjduffin.risk.usecase.request;

public class AttackRequest extends Request {
    private String player;
    private String attacker;
    private String defender;

    public AttackRequest(String player, String attacker, String defender) {
        this.requestType = Type.ATTACK;
        this.player = player;
        this.attacker = attacker;
        this.defender = defender;
    }

    public String getPlayer() {
        return player;
    }

    public String getAttacker() {
        return attacker;
    }

    public String getDefender() {
        return defender;
    }

}

File: risk-lib/src/main/java/net/mjduffin/risk/lib/usecase/request/AttackRequest.kt
package net.mjduffin.risk.lib.usecase.request

import net.mjduffin.risk.lib.usecase.request.Request

class AttackRequest(player: String, attacker: String, defender: String) : Request() {
    val player: String
    val attacker: String
    val defender: String

    init {
        requestType = Type.ATTACK
        this.player = player
        this.attacker = attacker
        this.defender = defender
    }
}